CONTRIBUTING.md

```diff
@@ -29,6 +29,13 @@ $ source .env/bin/activate
 (.env)$ pip install -r requirements.txt
 ```
 
+Or if you're on windows in a powershell window with py launcher installed:
```

Comment:
Great addition @Mark-Giesen! Windows development is a bit out of our view so it is great you are helping other Windows devs!!